clients/cloud/python/consumer.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+#
+# Copyright 2018 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# =============================================================================
+#
+# Consume messages from Confluent Cloud
+# Using Confluent Python Client for Apache Kafka
+#
+# =============================================================================
+
+from confluent_kafka import Consumer, KafkaError
+import sys
+import uuid
+import json
+
+
+if __name__ == '__main__':
+    if len(sys.argv) != 3:
+        sys.stderr.write('Usage: %s <path to Confluent Cloud configuration file> <topic name>\n' % sys.argv[0])
+        sys.exit(1)
+
+    # Read Confluent Cloud configuration for librdkafka clients
+    config_file = sys.argv[1]
+    conf = {}
+    with open(config_file) as fh:
+      for line in fh:
+        line = line.strip()
+        if line[0] != "#" and len(line) != 0:
+          parameter, value = line.strip().split('=', 1)
+          conf[parameter] = value.strip()
+
+    # Create Consumer instance
+    c = Consumer({
+        'bootstrap.servers': conf['bootstrap.servers'],
+        'broker.version.fallback': '0.10.0.0',
+        'api.version.fallback.ms': 0,
+        'sasl.mechanisms': 'PLAIN',
+        'security.protocol': 'SASL_SSL',
+        'sasl.username': conf['sasl.username'],
+        'sasl.password': conf['sasl.password'],
+        'group.id': str(uuid.uuid1()),  # this will create a new consumer group on each invocation.
```

Comment:
A real application would basically never do that, right? so I'm wondering why do we do this in an example. If we want to give the users an easy way to start from scratch, wouldn't a command-line parameter be a better option?

Reply:
+1, we do this in integration tests but it should not be used in the real world. Examples should be something you can essentially copy and paste.

Reply:
@rnpridgeon I agree with "Examples should be something you can essentially copy and paste."

And this is straight from the Confluent Kafka Python examples:

https://github.com/confluentinc/confluent-kafka-python/blob/34baea620f063cb96fc32b5a562170f19a19228e/examples/confluent_cloud.py#L89

I can remove it from this example

Reply:
Good catch, will fix